tests/integration/test_latex_exporting.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from datetime import date
+
+from inspirehep.utils.latex import Latex
+from inspirehep.utils.record_getter import get_db_record
+
+import pytest
+
+
+@pytest.mark.xfail(reason='wrong output')
+def test_format_latex_eu(app):
+    article = get_db_record('literature', 4328)
+    today = date.today().strftime('%d %b %Y')
```

Comment:
Mocking system time is actually a little trickier than it looks because most data structures that hold the time are immutable. It can be done, like I did for unit tests, but I'd rather not put too much mocking/logic inside of an integration test. I'm already dissatisfied for having to do `+ today`...